polaris-patterns/src/components/DatePickerTextField/DatePickerTextField.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {useState} from 'react';
```

Comment:
Copied from https://web.docs.shopify.io/components/Forms/DatePickerTextField. I did add a new prop for `datePlaceholder` since the web version uses translations for `YY-MM-DD`. Should we allow translations for this new `polaris-patterns` library or make our consumers pass strings in as props? The disadvantage to the prop solution is that we might get a lot of wrapper components like the above DatePicker